config/polygon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "network": "matic",
+  "geyser_factory_v1": "0x0000000000000000000000000000000000000000",
+  "geyser_factory_v1_start_block": 16009661,
+  "pool_factory": "0x0000000000000000000000000000000000000000",
+  "pool_factory_start_block": 16009661,
+  "gysr_address": "0xbea98c05eeae2f3bc8c3565db7551eb738c8ccab",
+  "weth_address": "0x7ceb23fd6bc0add59e62ac25578270cff1b9f619",
+  "usdc_address": "0x2791bca1f2de4661ed88a30c99a7a9449aa84174",
+  "usdt_address": "0xc2132d05d31c914a87c6611c10748aeb04b58e8f",
+  "dai_address": "0x8f3cf7ad23cd3cadbd9735aff958023239c6a063",
+  "wbtc_address": "0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6",
+  "uni_address": "0x831753dd7087cac61ab5644b308642cc1c33dc13",
+  "link_address": "0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39",
+  "bnb_address": "0x0000000000000000000000000000000000000000",
+  "snx_address": "0x0000000000000000000000000000000000000000",
+  "yfi_address": "0x0000000000000000000000000000000000000000",
+  "comp_address": "0x0000000000000000000000000000000000000000",
+  "mkr_address": "0x0000000000000000000000000000000000000000",
+  "grt_address": "0x0000000000000000000000000000000000000000",
+  "usdt_weth_address": "0xf6422b997c7f54d1c6a6e103bcb1499eea0a7046",
+  "uniswap_factory": "0x5757371414417b8C6CAad45bAeF941aBc7d3Ab32",
+  "sushi_factory": "0x5757371414417b8C6CAad45bAeF941aBc7d3Ab32",
```

Comment:
sushi is actually on polygon so we can use that address:
```
0xc35DADB65012eC5796536bD9864eD8773aBc74C4
```

Reply:
Nice, good call